### DISCUSSION TITLE
Gasless setup

### DISCUSSION THREAD

**Discussion by JanBajecDev** [ethereum-optimism/developers#523] [Application Development]
Has anybody set up gasless op-stack chain? ( so that gasPrice is always 0 for users )

**Comment by opfocus** [comment]
CC @sbvegan

**Comment by Chomtana** [comment]
I'm not sure whether this can be done. But if you want to try, at least, you need to modify the L1 fees submitted on each block to 0.

https://github.com/ethereum-optimism/optimism/blob/79babb8944e21f313e6ecf581210f237e0d5f6f7/op-node/rollup/derive/l1_block_info.go#L264-L323

Must set `BaseFee` and `BlobBaseFee` property of `l1BlockInfo` to 0

And edit the minimum gas price assertion to allow the sequencer to accept any txs which I don't sure where
